src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -28,7 +29,7 @@ public void Success()
             Logger.InfoFormat("The circuit breaker for {0} is now disarmed", name);
         }
 
-        public void Failure(Exception exception)
+        public Task Failure(Exception exception)
```

Comment:
@SeanFeldman I think we talked enough about that, you were also part of that discussion. Our decision was: On the move towards the end goal we postfix code which is directly used by customers with `Async`. In the end goal we will remove those postfixes